store/sqlstore/user_store.go

```diff
@@ -894,7 +852,6 @@ func (us SqlUserStore) GetProfileByIds(userIds []string, options *store.UserGetB
 
 		cpy := &model.User{}
```

Comment:
You don't need that `cpy` variable because you are no loger using it.

Reply:
Ok